src/forms/_base.scss

```diff
@@ -57,6 +57,10 @@ input[type="radio"] {
   -webkit-appearance: radio;
   border-radius: 50%;
   border: 1px solid color(text);
+
+  &:checked {
+    background-color: color(gray);
```

Comment:
is `gray` the right color to use? i think this just uses the "regular" browser `gray`.  unless that's what we want